README.md

```diff
@@ -1 +1,95 @@
-# ArachneCentralAPI
\ No newline at end of file
+#Arachne Community Edition build and run manual
+
+##Prerequisites
+For building and run the Arachne please install following applications:
+- Maven 
```

Comment:
Pls add versions and direct links to download where it's possible

Reply:
done